docs/relay.md

```diff
@@ -142,11 +142,7 @@ const userType = new GraphQLObjectType({
     name: {
       type: GraphQLString
     },
-    tasks: {
-      type: userTaskConnection.connectionType,
-      args: userTaskConnection.connectionArgs,
-      resolve: userTaskConnection.resolve
-    }
+    tasks: userTaskConnection.field, /* {type, args, resolve} * /
```

Comment:
i just noticed that the old style is no longer documented, which it absolutely should be, so please restore the original documentation and then add additions as new pieces of documentation.

Reply:
You know, I've found in my own work that this PR isn't really as useful as I had first imagined.  maybe it will just add confusion.  Do you think I should finish up the documentation or should we just close this?

Reply:
Fine to close by me.